pipeline/load.py

```diff
@@ -27,7 +28,7 @@ def get_connection(host: str, db_name: str, password: str, user: str):
         print(f"Error {e} occured!")
 
 
-def switch_between_schemas(schema_name: str) -> None:
+def switch_between_schemas(conn, schema_name: str) -> None:
```

Comment:
you could also put the type hint in here: 
```suggestion
def switch_between_schemas(conn: connection, schema_name: str) -> None:
```